app/models/file_search.rb

```diff
@@ -43,24 +61,28 @@ def venue_names
     params.fetch(:venues) { all_venue_names }
   end
 
-  def record_types
-    params.fetch(:record_types) { all_record_types }
+  def resource_types
+    params.fetch(:resource_types) { all_resource_types }
+  end
+
+  def types
+    params.fetch(:types) { all_resource_types }
```

Comment:
Is `types` a mutliple thing?  I thought we were just querying for one type at a time.